python/api/main.py

```diff
@@ -167,4 +167,36 @@ async def get_percentile_circles(stop_id: str):
             cur.execute("SELECT api.get_percentile_circles_with_stop_id(%(stop_id)s)", {'stop_id': stop_id })
             percentile_circles = cur.fetchall()
 
-        return tuples_to_feature_collection(geom_tuples=percentile_circles)
\ No newline at end of file
+        return tuples_to_feature_collection(geom_tuples=percentile_circles)
+
+@app.get("/monitored_vehicle_journeys")
+async def get_monitored_vehicle_journeys(operating_day: str):
+    """
+    Returns assumed monitored vehicle journeys from given operating day. Assumed here means
+    that the journeys might be valid or not, API doesn't know it. Invalid journey is example
+    a journey where bus driver signed in to a wrong departure.
+    """
+    with psycopg.connect(get_conn_params()) as conn:
+        with conn.cursor() as cur:
+            cur.execute("SELECT * FROM api.view_assumed_monitored_vehicle_journey where oday = %(operating_day)s", {'operating_day': operating_day})
+            vehicle_journeys = cur.fetchall()
+
+            results = []
```

Comment:
There's a [way](https://varun-verma.medium.com/use-psycopg2-to-return-dictionary-like-values-key-value-pairs-4d3047d8de1b) to return the result set from the database directly as a collection of dicts, meaning we have the column names available right away and don't have to create them "by hand". Oddly enough, this is not the default way of psycopg2 but needs importing its extras module.

This current way is OK and clear to read and understand 👍  As more API endpoints are created, though, maybe we can consider using the Dict cursor method then to make the Python code more concise.